assets/stylesheets/transactions/transactions.css

```diff
@@ -0,0 +1,21 @@
+#transactions{
+	position: relative;
+	position: center;
+	background-color: #00000;
+}
+#main{
```

Comment:
start the styling from the outermost element
`main `goes first and then its child elements `table`, etc.